### PR TITLE
Fix gascalibration cmd name

### PR DESCRIPTION
**Pull request by ivange94**
# Description of change

Fix gascalibration cmd generation

### CODE REVIEW

File: contracts/wasm/gascalibration/gascalibration/main.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+)
+
+var (
+	dir string
+
+	rootCmd = &cobra.Command{
+		Version: wasp.Version,
+		Use:     "gascalibration",
+		Args:    cobra.NoArgs,
+		Short:   "gascalibration is a command line tool for generate gas calibration reports.",
```
Comment (lmoe) [inline contracts/wasm/gascalibration/gascalibration/main.go]:
```suggestion
		Short:   "gascalibration is a command line tool to generate gas calibration reports.",
```

File: contracts/wasm/gascalibration/gascalibration/main.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		Long:    `gascalibration is a command line tool for generate gas calibration reports from storage, memory and execution time contracts.`,
```
Comment (lmoe) [inline contracts/wasm/gascalibration/gascalibration/main.go]:
```suggestion
		Long:    `gascalibration is a command line tool to generate gas calibration reports from storage, memory and execution time contracts.`,
```

File: contracts/wasm/gascalibration/gascalibration/main.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+		err := plotutil.AddLinePoints(p, v...)
+		check(err)
+
+		filePath := path.Join(dir, contract+".png")
+		err = p.Save(8*vg.Inch, 8*vg.Inch, filePath)
+		check(err)
+	}
+}
+
+func graphData(filename string, points map[uint32]uint64) plotter.XYs {
```
Comment (lmoe) [inline contracts/wasm/gascalibration/gascalibration/main.go]:
```suggestion
func graphData(_ string, points map[uint32]uint64) plotter.XYs {
```

File: contracts/wasm/gascalibration/gascalibration/main.go [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+	_ = filename
```
Comment (lmoe) [inline contracts/wasm/gascalibration/gascalibration/main.go]:
```suggestion
```

File: contracts/wasm/gascalibration/gascalibration/main.go [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+		return "Golang"
+	} else if strings.Contains(filename, "rs") {
+		return "Rust"
+	} else if strings.Contains(filename, "ts") {
+		return "Typescript"
+	}
+	return "Solidity"
+}
+
+func check(err error) {
```
Comment (lmoe) [inline contracts/wasm/gascalibration/gascalibration/main.go]: I think `log.Check` already does what you have established with your own `check` function.

**Review by lmoe — APPROVED** [review]

File: contracts/wasm/gascalibration/gascalibration/main.go (hunk 5 above)
Comment (ivange94) [reply contracts/wasm/gascalibration/gascalibration/main.go]: Nice catch. Actually the earlier code didn't use `log.Check`. I only refactored it to use it in this commit but didn't realize it's now redundant.

**Review by dessaya — APPROVED** [review]

**Pull request merged**